en/docs/how-tos/security/transport-security.md

```diff
@@ -24,3 +24,37 @@ The following are the default configurations for a gateway instance. Add the fol
 |sslVerifyClient|The type of client certificate verification. ("require" or "optional")| "optional"|
 
 
+### Mutual SSL with Backend 
+
+!!! Note
+    This feature is only available with the WUM updates and is effective from 7th October 2020 (2020-10-07) for Microgateway Toolkit 3.2.0. 
+
+When the API Backend is secured with Mutual SSL, the Micro Gateway HTTP client should be configured to support mutual ssl. 
+
+To do this, add the following configuration to the [httpClients] section in micro-gw.conf file.
+
+``` yaml tab="micro-gw.conf"
```

Comment:
Isn't this enabled by default

Reply:
Yes. The instructions should be changed. 
Fixed in https://github.com/wso2/docs-mg/pull/156/commits/ac94c99cba7a866d9845ccd6c51330a49bba1bc5
Thanks